index.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html>
+<head>
+      <title>Portfolio</title>
+	  <meta charset="UTF-8">
+	  <meta name="viewport" content="width=device-width, initial-scale=1">
+	  <link rel="stylesheet" type="text/css" href="styles.css">
+	  <link href="https://fonts.googleapis.com/css?family=Ubuntu" rel="stylesheet" type="text/css">
+	  <link rel="stylesheet" type="text/css" href="normalize.css">
+</head>
+<body>	
+	<nav>
+		<div class="initials">
+			<h3 class="name">Yasmin Shenair</h3>
+			<a href="#" class="icon" title="initials">
+				<img src="ys.png"alt="initials" />
+			</a>
+			<a href="#" class="icon" title="icon">
+				<img src="hamburger.png"alt="hamburger-menu" />
```

Comment:
same again here - needs a space between 'src' and 'alt' attributes

Reply:
And done lol..